installation/resources/installer-cr.yaml.tpl

```diff
@@ -62,5 +62,5 @@ spec:
     #  namespace: "kyma-system"
     #- name: "jaeger"
     #  namespace: "kyma-system"
-    #- name: "monitoring"
-    #  namespace: "kyma-system"
+    - name: "monitoring"
```

Comment:
you are changing the default installation behavior of kyma. For an optional feature. I think that should be solved by documentation instead